ssh/foxpass-keygen.py

```diff
@@ -46,14 +46,14 @@ def main():
 
     email = input('Email address: ')
     password = getpass.getpass()
-    mfa_code = input('Foxpass MFA code:')
+    mfa_code = input('Foxpass MFA code: ')
 
     try:
         add_key_to_foxpass(api_base, email, password, mfa_code, filename)
         print('')
-        print('Don\'t forget to run ssh-add now. The -K adds the key to your keychain so it is loaded on boot:')
+        print('Don\'t forget to run ssh-add now. The --apple-use-keychain flag (OS X only) adds the key to your keychain so it is loaded on boot:')
         print('')
-        print('ssh-add -K {}'.format(filename))
+        print('ssh-add --apple-use-keychain {}'.format(filename))
```

Comment:
This is still -K on my mac (Big Sur).

I guess it changed in Mavericks?

```
$ ssh-add -help
ssh-add: illegal option -- h
usage: ssh-add [options] [file ...]
Options:
  -l          List fingerprints of all identities.
  -E hash     Specify hash algorithm used for fingerprints.
  -L          List public key parameters of all identities.
  -k          Load only keys and not certificates.
  -c          Require confirmation to sign using identities
  -m minleft  Maxsign is only changed if less than minleft are left (for XMSS)
  -M maxsign  Maximum number of signatures allowed (for XMSS)
  -t life     Set lifetime (in seconds) when adding identities.
  -d          Delete identity.
  -D          Delete all identities.
  -x          Lock agent.
  -X          Unlock agent.
  -s pkcs11   Add keys from PKCS#11 provider.
  -e pkcs11   Remove keys provided by PKCS#11 provider.
  -T pubkey   Test if ssh-agent can access matching private key.
  -q          Be quiet after a successful operation.
  -A          Add all identities stored in your keychain.
  -K          Store passphrases in your keychain.
              With -d, remove passphrases from your keychain.
```

Reply:
```
❯ ssh-add -K ~/.ssh/foxpass-ssh-20160412-120639
WARNING: The -K and -A flags are deprecated and have been replaced
         by the --apple-use-keychain and --apple-load-keychain
         flags, respectively.  To suppress this warning, set the
         environment variable APPLE_SSH_ADD_BEHAVIOR as described in
         the ssh-add(1) manual page.
```